mibios/glamr/views.py

```diff
@@ -493,14 +501,86 @@ def get_details(self):
             pk = 0
         else:
             pk = self.object.pk
-        # prepend samples row
-        details = [(
-            'Samples',
-            reverse('dataset_sample_list', args=[pk]),
-            f'List of {self.object.samples().count()} samples'
-        )] + details
-        return details, rel_lists
 
+        # this is hacky but want details in particular order
+        ordered_details = []
+        # reference, samples, bioproject, gold id, material type, water bodies,
+        # primers, sequencing target, sequencing platform, size fraction, notes
```

Comment:
I've just commited to main branch a field ordering for the detail branch, so just set field_order = [...]